be/src/vec/functions/function_hash.h

```diff
@@ -20,14 +20,6 @@
 
 #pragma once
 
-#include "vec/common/hash_table/hash.h"
```

Comment:
why not delete this file

Reply:
```hash.h``` is used by IntHash64Impl. 
```hash.h``` contains int_hash64.